### ISSUE TITLE
Update for Cypress 10.x+

### ISSUE THREAD

**Issue by kamranayub** [kamranayub/cypress-browser-permissions#53]
I do not think this works with Cypress@10 or above due to the changes in folder structure, or at least the instructions need to be updated.

**Comment by MikeMcC399** [comment]
@kamranayub 

Are you intending to maintain this plugin? Apart from support for Cypress `13.x`,  it would need workflows updating to current versions in https://github.com/kamranayub/cypress-browser-permissions/tree/master/.github/workflows.

**Comment by kamranayub** [comment]
Yes, I have a demo sample that's still using Cypress 4 that needs updating. I think for my app I'm using Cypress 12 but I'm not using this plugin yet for that.
